tests/test_plotting.py

```diff
@@ -128,5 +184,15 @@ def test_legend(self):
         self.assertEqual(labels, expected)
 
 
+def _check_colors(collection, expected_colors):
+
+    import matplotlib.colors as colors
+    conv = colors.colorConverter
+
+    for patch, color in zip(collection, expected_colors):
+        result = patch.get_color()
+        assert conv.to_rgba(result) == conv.to_rgba(color)
```

Comment:
Looks like the build is failing here after merging to master.

This probably should be `numpy.testing.assert_almost_equal()` instead of a direct equality check, but the difference currently is bigger tha would make a difference for that.

The test failure: https://travis-ci.org/geopandas/geopandas/jobs/88372548#L583